rabbitmq/auto_conf/rabbitmq.yaml

```diff
@@ -0,0 +1,68 @@
+docker_images:
+  - rabbitmq
```

Comment:
As we are adding test data, please give the sd ID a more specific name (as in `rabbitmq-testqueues`) to differenciate from other flavors that could be added later